engine/render/src/render/font_renderer.cpp

```diff
@@ -562,6 +563,16 @@ namespace dmRender
         te.m_SourceBlendFactor = params.m_SourceBlendFactor;
         te.m_DestinationBlendFactor = params.m_DestinationBlendFactor;
 
+        // find center and radius for frustum culling
+        dmVMath::Vector4 cornerpoint_world = params.m_WorldTransform.getCol(3); // origin of label is always its bottom left. 'cornerpoint_world' is where it ends up after transforms
+        dmVMath::Vector4 centerpoint_local(params.m_Width/2, params.m_Height/2, 0, 1); // center of label cell in local coords
+        dmVMath::Vector4 centerpoint_world = params.m_WorldTransform * centerpoint_local; // transform to world coordinates
+
+        te.m_FrustumCullingRadius = Vectormath::Aos::length(cornerpoint_world - centerpoint_world);
+        te.m_FrustumCullingCenter.setX(centerpoint_world.getX());
+        te.m_FrustumCullingCenter.setY(centerpoint_world.getY());
+        te.m_FrustumCullingCenter.setZ(centerpoint_world.getZ());
```

Comment:
As discussed offline, we should do a pass of the rendering algorithm to produce an accurate bounding box, to make sure the users text is showing.

Reply:
I've supplied new algorithm. Let's comment on the new implementation.

Marking this as resolved.